AnkiDroid/src/main/java/com/ichi2/compat/CompatV26.java

```diff
@@ -183,4 +186,44 @@ public void abandonAudioFocus(AudioManager audioManager, AudioManager.OnAudioFoc
             audioManager.abandonAudioFocusRequest(audioFocusRequest);
         }
     }
+
+    /*
+     * This method uses [Files.newDirectoryStream].
+     * Hence this method, hasNext and next should be constant in time and space.
+     */
+    @Override
+    public @Nullable FileStream contentOfDirectory(File directory) {
+        DirectoryStream<Path> paths_stream;
+        try {
+            paths_stream = Files.newDirectoryStream(directory.toPath());
+        } catch (IOException e) {
+            Timber.w(e);
+            return null;
+        }
+        Iterator<Path> paths = paths_stream.iterator();
+        return new FileStream() {
+            @Override
+            public void close() throws IOException {
+                paths_stream.close();
+            }
+
+
+            @Override
+            public boolean hasNext() throws IOException{
```

Comment:
interesting - how did this spacing thing slip through ktlint :thinking: ?

```suggestion
            public boolean hasNext() throws IOException {
```

Reply:
Because it's a java file

Reply:
oh of course :facepalm: